README.md

```diff
@@ -3,3 +3,27 @@
 # CMIP6 Hackathon Main Website
 
 Website: [cmip6hack/cmip6hack.github.io](https://cmip6hack.github.io)
+
+
+# To build and test locally
+
+Create a conda environment
+```bash
+conda create -n web-dev -c conda-forge nodejs
```

Comment:
There are some pre-commit hooks for this repository. Can we update 

```bash
conda create -n web-dev -c conda-forge nodejs
conda activate web-dev
```
with 

```bash
conda create -n web-dev -c conda-forge nodejs pre-commit
conda activate web-dev
pre-commit install
```

Reply:
pre-commit install didn't work for me

Reply:
> pre-commit install didn't work for me

Interesting... What was the error?

Reply:
pre-commit: command not found